client/me/reauth-required/index.jsx

```diff
@@ -320,5 +316,8 @@ export default connect(
 		currentUser: getCurrentUser( state ),
```

Comment:
The `currentUser` prop is no longer used after this.

Reply:
Of course. Removed in https://github.com/Automattic/wp-calypso/pull/53661/commits/fde89995cc649b0b571935233926eef07e17dacc